api/browser.ts

```diff
@@ -8,6 +8,12 @@ import {
 } from "@/api/message";
 import { ScriptOption } from "@/lib/wallet/wallet-interface.ts";
 import { EthereumBrowserAPI } from "./ethereum";
+import { ApiUtils } from "@/api/background/utils.ts";
+
+window.postMessage(
+  ApiUtils.createApiResponse("kastle_installed", []),
+  window.location.origin,
+);
```

Comment:
It is for sdk, right? Maybe it would be better to put inside the constructor of BrowserAPI